docs/guides/8-continuous-integration.md

```diff
@@ -77,4 +77,4 @@ The `-f` (format) flag is used in the script to pick the JUnit output format. Th
 
 ## Jenkins
```

Comment:
Ugh. I wonder how long we've said soon... Maybe just remove this or suggest that you can use Jenkins and leave it at that?